css/selectors/nesting.json

```
@@ -0,0 +1,41 @@
+{
+  "css": {
+    "selectors": {
+      "nesting": {
+        "__compat": {
+          "description": "Nesting selector (<code>&</code>)",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Nesting_selectors",
+          "spec_url": "https://www.w3.org/TR/css-nesting-1/#nesting-selector",
+          "support": {
+            "chrome": {
+              "version_added": "112"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "16.5"
```

This should do the trick!

```suggestion
              "version_added": "preview"
```